awx/ui/apps.py

```diff
@@ -1,9 +0,0 @@
-# Django
```

Comment:
This couples with the configuration system in a few ways. You may end up needing to bring one or both of these files over as well.